doc/OnlineDocs/getting_started/index.rst

```diff
@@ -1,9 +1,58 @@
 Getting Started
 ===============
 
-Installation, pyomo command, a simple example, etc
+Installation
+------------
 
-.. doctest::
+Pyomo currently supports the following versions of Python:
+
+* CPython: 2.7, 3.5, 3.6
+
+
+Using CONDA
+~~~~~~~~~~~
+
+We recommend installation with *conda*, which is included with the Anaconda
```

Comment:
I do not recommend conda.  I am still confused as to why we don't list pip as the standard install route for pyomo and list conda as a convenience for Anaconda users.  Recommending miniconda for non-conda platforms results in a new Python on their system, which will cause problems with IDEs, PATHs, etc.

Reply:
I do recommend conda, because pyomo.extras robustly installs third-party packages.  We tried using a similar package for PyPI and users had problem with package installations.

You don't need to install miniconda to use conda.